tests/helpers.py

```diff
@@ -303,6 +303,25 @@ def __init__(self, roles: Optional[Iterable[MockRole]] = None, **kwargs) -> None
             self.mention = f"@{self.name}"
 
 
+# Create a User instance to get a realistic Mock of `discord.User`
+user_instance = discord.User(data=unittest.mock.MagicMock(), state=unittest.mock.MagicMock())
+
+
+class MockUser(CustomMockMixin, unittest.mock.Mock, ColourMixin, HashableMixin):
+    """
+    A Mock subclass to mock User objects.
+
+    Instances of this class will follow the specifications of `discord.User` instances. For more
+    information, see the `MockGuild` docstring.
+    """
+    def __init__(self, **kwargs) -> None:
+        default_kwargs = {'name': 'user', 'id': next(self.discord_id), 'bot': False}
+        super().__init__(spec_set=user_instance, **collections.ChainMap(kwargs, default_kwargs))
+
+        if 'mention' not in kwargs:
+            self.mention = f"@{self.name}"
```

Comment:
Shouldn't this be `<@id>`? That's how discord.py does it.

Reply:
I can change it; I used this because `name` is usually more descriptive in the test output and I did not think having the actual mention format here would aid in testing. If a particular piece of code does need the correct format, an explicit mention can still be passed as a kwarg.

Reply:
Nah it's fine.